### DISCUSSION TITLE
No device to select in the blueprint

### DISCUSSION THREAD

**Discussion by josew029** [IATkachenko/HA-SleepAsAndroid#70] [Q&A] [answered]
Hello, 

I have a similar problem as described here: https://github.com/IATkachenko/HA-SleepAsAndroid/discussions/41
I don't get a device displayed in the blueprint either, not even mosquitto-broker.

But unfortunately I still haven't reached my goal. First of all, I'm still completely new to HA. 

About my settings: 
In the Sleep as Android:
> MQTT activated
> URL: tcp://mqtt-user:password@192.168.xxx.xx:1883
> Topic: SleepAsAndroid/myPixel
> Client ID: AleepAsAndroid

When I tap on Test: ‘Connecting...’ and then ‘Success’

The MQTT Explorer displays the following:
> 192.168.xxx.xx
> $SYS
> SleepAsAndroid
> myPixel = {‘event’: ‘Unknown’}

Hope u cant help me

Thanks so far.

SleepAsAndroid v20240701
HomeAssistant v2024.8.3

**Comment by IATkachenko** [comment]
@josew029, hello.

Your application configuration looks well.
Let's check HA now.

Is HA creates device in the integration?
What topic string do you use in the integration? It should be `SleepAsAndroid/myPixel` or `SleepAsAndroid/%%%device%%%`.

**Reply by josew029** [reply, marked as answer]
@IATkachenko thx for the fast replay. 

> What topic string do you use in the integration? It should be SleepAsAndroid/myPixel or SleepAsAndroid/%%%device%%%.

This was the right hint. The topic name in the integration was wrong :) 
Obviously this should be also SleepAsAndroid/myPixel
